src/mito2/src/sst/parquet/writer.rs

```diff
@@ -47,11 +51,13 @@ impl ParquetWriter {
     /// Creates a new parquet SST writer.
     pub fn new(
         file_path: String,
+        index_file_path: String,
```

Comment:
non-blocking: What about setting it via methods like `with_index_file_path` instead of a required argument to `new()`.